docs/reference/data-management.asciidoc

```diff
@@ -0,0 +1,33 @@
+[role="xpack"]
+[[data-management]]
+= Data management
+
+[partintro]
+--
+The data you store in {es} generally falls into one of two categories:
+
+* A collection of content you want to search such as a catalog of products
+* A stream of continuously-generated time series data such as log entries
+
+Content collections might be frequently updated, 
+but the value of the content remains relatively constant over time. 
+You want to be able to retrieve items quickly regardless of how old they are. 
+
+Time series data just keeps accumulating over time, so you need strategies for 
+balancing the value of the data against the cost of storing it. 
+As it ages, it tends to become less important and less-frequently accessed, 
+so you can move it to less expensive, less performant hardware. 
```

Comment:
```suggestion
so we can move it to less expensive, less performant hardware. 
```

Reply:
We generally avoid "we" in favor of "you" unless it's in the context of a recommendation.